inference-engine/tests/functional/plugin/cpu/single_layer_tests/tile.cpp

```diff
@@ -0,0 +1,323 @@
+                result << CommonTestUtils::vec2str(item) << "_";
+            }
+        }
+        result << "Repeats=" << CommonTestUtils::vec2str(repeats)  << "_";
+        result << "netPrec=" << netPrecision << "_";
+        result << "constRepeats=" << (isRepeatsConst ? "True" : "False") << "_";
+        result << "trgDev=" << deviceName;
+
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        TileLayerTestParamsSet basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+
+        std::vector<ov::test::InputShape> inputShapes;
+        ov::element::Type_t netPrecision;
+        bool isRepeatsConst;
+        std::tie(inputShapes, repeatsData, netPrecision, isRepeatsConst, targetDevice) = basicParamsSet;
+
+        selectedType += std::string("_") + ov::element::Type(netPrecision).get_type_name();
+
+        if (inputShapes.front().first.rank() != 0) {
+            inputDynamicShapes.push_back(inputShapes.front().first);
+            if (!isRepeatsConst) {
+                inputDynamicShapes.push_back({ static_cast<int64_t>(repeatsData.size()) });
+            }
+        }
+        const size_t targetStaticShapeSize = inputShapes.front().second.size();
+        targetStaticShapes.resize(targetStaticShapeSize);
+        for (size_t i = 0lu; i < targetStaticShapeSize; ++i) {
+            targetStaticShapes[i].push_back(inputShapes.front().second[i]);
+            if (!isRepeatsConst)
+                targetStaticShapes[i].push_back({ repeatsData.size() });
+        }
+
+        ov::ParameterVector functionParams;
+        if (inputDynamicShapes.empty()) {
+            functionParams.push_back(std::make_shared<ov::op::v0::Parameter>(netPrecision, targetStaticShapes.front().front()));
+        } else {
+            functionParams.push_back(std::make_shared<ov::op::v0::Parameter>(netPrecision, inputDynamicShapes.front()));
+            if (!isRepeatsConst) {
+                functionParams.push_back(std::make_shared<ov::op::v0::Parameter>(ov::element::i64, inputDynamicShapes[1]));
+                functionParams.back()->set_friendly_name("repeats");
+            }
+        }
+        functionParams.front()->set_friendly_name("data");
+
+        auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ov::op::v0::Parameter>(functionParams));
+        std::shared_ptr<ov::Node> tileNode;
+        if (isRepeatsConst) {
+            tileNode = std::make_shared<ov::op::v0::Tile>(paramOuts[0],
+                    ov::op::v0::Constant::create(ov::element::i64, { repeatsData.size() }, repeatsData));
+        } else {
+            tileNode = std::make_shared<ov::op::v0::Tile>(paramOuts[0], paramOuts[1]);
+        }
+        tileNode->get_rt_info() = getCPUInfo();
```

Comment:
Please remove this line, `getCPUInfo()` is called inside `makeNgraphFunction`

Reply:
Removed.